client/src/main/kotlin/io/spine/examples/shareaware/client/Client.kt

```diff
@@ -293,18 +293,22 @@ public class DesktopClient private constructor(
  * @param entityType type of the entity on which changes subscription works
  * @param client client with the help of which the entity will be subscribed
  * @param id entity ID by which arrived entities will be filtered
+ * @param beforeUpdate the callback function that provides access to the state of the entity
```

Comment:
Honestly, I don't understand what this thing does.

Please consider adding more details into the body of the documentation (not to `@param` section), which would illustrate the idea.

Right now I think its name does not reflect the functionality. `beforeUpdate` is something that is invoked before the update operation initiated on our end, begins.